framework/generated/vulkan_generators/base_generator.py

```diff
@@ -71,6 +71,7 @@ def _make_re_string(list, default=None):
 _remove_extensions = [
     "VK_KHR_video_queue", "VK_KHR_video_decode_queue",
     "VK_KHR_video_encode_queue", "VK_EXT_video_encode_h264",
+    'VK_KHR_video_decode_h264', 'VK_KHR_video_decode_h265', 
```

Comment:
We should have issues for adding them which reference this PR and the header version that introduced them.

Reply:
Added https://github.com/LunarG/gfxreconstruct/issues/956